src/main/java/io/cdap/plugin/salesforce/SalesforceConnectionUtil.java

```diff
@@ -41,7 +41,7 @@ public class SalesforceConnectionUtil {
    */
   public static PartnerConnection getPartnerConnection(AuthenticatorCredentials credentials)
```

Comment:
make sure we are not calling this anywhere in record reader

Reply:
Being used in soapRecReader and WideRecReader